packages/members-api/index.js

```diff
@@ -222,6 +223,47 @@ module.exports = function MembersApi({
         return magicLinkService.sendMagicLink({email, type, requestSrc, tokenData: Object.assign({email}, tokenData)});
     }
 
+    /** Destructive method alert: This will clear all Stripe related table data from DB, is used for
```

Comment:
IMO This comment isn't useful, there's no guarantee someone using this method will see it. Probably better off calling the method `deleteAllStripeData` so it's very clear how dangerous it is